cycledash/templates/comments.html

```diff
@@ -0,0 +1,13 @@
+{% extends "layouts/layout.html" %}
```

Comment:
Is it good practice to have a template with the same name as a macro? Seems confusing to me. Maybe name the macro `comment_list`?